### PR TITLE
Add sidebar footer

### PR DESCRIPTION
**Pull request by tatianainama**
This PR adds the sidebar footer.
I decided to not swizzle the DocSidebar component but extend it to avoid breaking default functionality.
I've also made the category item (the first of each list of links) [expanded by default](https://github.com/GetStream/stream-chat-docusaurus-cli/pull/14/files#diff-b9033b845a7bc093a7c500d41642fea3547e3c9d714a0eaff13f0da38388b189R12)

### CODE REVIEW

File: docusaurus/src/theme/DocSidebar.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+const WEB_LINKS = [
+  ['Contact Support', 'https://getstream.io/contact/support/'],
+  ['Maker Account', 'https://getstream.io/maker-account/'],
+  ['Mobile Chat Kit', 'https://getstream.io/chat/ux-kit/']
+]
+
+export default function DocSidebar(props) {
+  return <>
+  <OriginalDocSidebar {...props} sidebar={props.sidebar.map(category => ({ ...category, collapsed: false }))}/>
```
Comment (tsirlucas) [inline docusaurus/src/theme/DocSidebar.js:12]: can you memoize the sidebar map logics? like: 
```
const sidebar = useMemo(() => props.sidebar.map(category => ({ ...category, collapsed: false })) , props.sidebar)
```

and then `sidebar={sidebar}`.

Just to make sure it wont run when its not needed

**Review by tsirlucas — CHANGES_REQUESTED** [review]

File: docusaurus/src/theme/DocSidebar.js (hunk 1 above)
Comment (tatianainama) [reply docusaurus/src/theme/DocSidebar.js]: Good catch!

File: docusaurus/src/theme/DocSidebar.js [hunk 2]
```diff
@@ -7,23 +7,25 @@ const WEB_LINKS = [
   ['Mobile Chat Kit', 'https://getstream.io/chat/ux-kit/']
 ]
 
-export default function DocSidebar(props) {
+export default function DocSidebar({ sidebar, ...props }) {
+  const sidebarItems = useMemo(() => sidebar.map(category => ({ ...category, collapsed: false  })));
```
Comment (tsirlucas) [inline docusaurus/src/theme/DocSidebar.js:11]: can you add `[sidebar]` as second parameter of `useMemo`?

Comment (tatianainama) [reply docusaurus/src/theme/DocSidebar.js]: Sure! sorry about that 😓

**Review by tsirlucas — APPROVED** [review]

**Pull request merged**